x/concentrated-liquidity/position.go

```diff
@@ -16,6 +16,9 @@ func (k Keeper) getOrInitPosition(
 	lowerTick, upperTick int64,
 	liquidityDelta sdk.Dec,
 ) (*model.Position, error) {
+	if !k.poolExists(ctx, poolId) {
```

Comment:
We can remove the todo from above the method. Please consider adding a simple godoc as well